modules/sdk/src/main/scala/org/tessellation/sdk/domain/statechannel/StateChannelValidator.scala

```diff
@@ -38,19 +39,21 @@ object StateChannelValidator {
     maxBinarySizeInBytes: Long = 500 * 1024
   ): StateChannelValidator[F] = new StateChannelValidator[F] {
 
-    override def validate(stateChannelOutput: StateChannelOutput): F[StateChannelValidationErrorOr[StateChannelOutput]] = for {
-      signaturesV <- signedValidator
-        .validateSignatures(stateChannelOutput.snapshotBinary)
-        .map(_.errorMap[StateChannelValidationError](InvalidSigned))
-      snapshotSizeV <- validateSnapshotSize(stateChannelOutput.snapshotBinary)
-      signaturesAllowedV = validateAllowedSignatures(stateChannelOutput)
-      genesisAddressV = validateStateChannelGenesisAddress(stateChannelOutput.address, stateChannelOutput.snapshotBinary)
-    } yield
-      signaturesV
-        .product(snapshotSizeV)
-        .product(signaturesAllowedV)
-        .product(genesisAddressV)
-        .as(stateChannelOutput)
+    def validate(stateChannelOutput: StateChannelOutput): F[StateChannelValidationErrorOr[StateChannelOutput]] =
+      validateHistorical(stateChannelOutput).map(_.product(validateAllowedSignatures(stateChannelOutput)).as(stateChannelOutput))
+
+    def validateHistorical(stateChannelOutput: StateChannelOutput): F[StateChannelValidationErrorOr[StateChannelOutput]] =
+      for {
+        signaturesV <- signedValidator
+          .validateSignatures(stateChannelOutput.snapshotBinary)
+          .map(_.errorMap[StateChannelValidationError](InvalidSigned))
+        snapshotSizeV <- validateSnapshotSize(stateChannelOutput.snapshotBinary)
```

Comment:
I wonder if we shouldn't also disable the size check for the historical snapshots in case it decreases in the future. Would be best to have the config per ordinal...

Reply:
I'd keep it for now as the change is purely about the seedlist (removal of nodes). We can discuss other changes on the meeting and introduce maybe that config per ordinal :+1: